content/en/docs/Concepts/MVC.md

```diff
@@ -8,11 +8,127 @@ description: >
 
-Aenean dui lectus, tincidunt eget mi ut, eleifend semper nunc. Suspendisse in sodales ligula, vitae auctor augue. Phasellus pellentesque eleifend auctor. Ut at eros odio. Aenean facilisis pretium sem, nec porttitor eros maximus dapibus. Aliquam erat volutpat. Ut a nisl tortor. Phasellus vehicula ac lacus in elementum. Nam nibh purus, hendrerit id vulputate sit amet, blandit id quam. Aenean cursus erat nisl, id maximus mauris ornare sed. Sed finibus aliquet imperdiet. Praesent lobortis sapien eget metus feugiat, ac pellentesque est porttitor. Donec congue sit amet lacus eu tempor.
+Because the MVC format is so separated, people can work on different sections of the code at the same time with very little complications occurring. This separation also makes debugging more simple.
+
+## Routing
+
+Routing in MVC usually follows a specific convention, but it can be done nearly any way by altering startup code. When making a project with MVC in Microsoft Visual Studio 2022 using ASP.NET Core 6, the default routing is the following:
+
+```csharp
+app.MapControllerRoute(
+    name: "default",
+    pattern: "{controller=Home}/{action=Index}/{id?}");
+
+```
+
+This code specifies the pattern of routing on startup. The name field allows you to define what route should be followed for different situations. The pattern field describes what comes after the default domain name in the link of a web application, and tells the program exactly what to do when given the information. The code shown also provides default values for the pattern. When using multiple routes, routing will often be defined as follows:
+
+```csharp
+app.UseEndpoints(endpoints =>
+{
+    endpoints.MapRazorPages();
+
+    endpoints.MapControllerRoute(
+        name: "default",
+        pattern: "{controller=Home}/{action=Index}/{id?}"
+    );
+});
+
+```
+
+This code also shows how endpoints can be used to determine what routing should be used.
+
+## Areas
+
+Areas are an organizational tool in MVC that allows programs to have much more effective routing and a set hierarchy to operate with. Areas are effectively a structure; thus, their content corresponds to a specific function or purpose in a program. For example, a program designed for  car dealership may have areas like "Rent," "Purchase," and "Sell," in order to reasonably separate the code. Areas are important in large programs in order to separate code in functional categories that may differ from simply their form (instead of grouping all controllers or all models, group their functions together).
```

Comment:
Extra space between for and car, also fix wording there too

Reply:
Emphasize the point of what an area actually is; A folder that contains controllers, models, and views all in one.